src/main/java/com/epam/machine/controller/ClientsInfoServlet.java

```diff
@@ -1,19 +1,39 @@
 package com.epam.machine.controller;
 
+import com.epam.machine.entity.Client;
+import com.epam.machine.service.ClientServiceImpl;
+
 import javax.servlet.ServletException;
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
+import javax.servlet.http.HttpSession;
 import java.io.IOException;
+import java.sql.SQLException;
 
 public class ClientsInfoServlet extends HttpServlet {
     @Override
     public void doGet (HttpServletRequest request, HttpServletResponse response) {
         try {
+            ClientServiceImpl clientServiceimpl = new ClientServiceImpl();
```

Comment:
It's better to define it at class level, we shouldn't create new instance of `ClientServiceImpl` in every method, also since we have interface for this class it's good practice to write it like this `ClientService clientService = new ClientServiceImpl();`